mivs/models.py

```diff
@@ -113,6 +113,15 @@ class IndieStudio(MagModel):
     email_model_name = 'studio'
 
     @property
+    def confirm_deadline(self):
```

Comment:
I opted to make the `accepted` property per-game but the `confirm_deadline` per-studio. I figured it'd be useful to track when each game is accepted, but for a more consistent UX we should just tell studios they have one deadline. As you can see, we use the first accepted game as the basis for the deadline.

Practically speaking, I imagine most games from a studio are accepted within the same day.